native/proto/src/proto/operator.proto

```diff
@@ -82,10 +82,18 @@ message Limit {
   int32 offset = 2;
 }
 
+enum CompressionCodec {
+  None = 0;
+  Zstd = 1;
+  Lz4 = 2;
```

Comment:
Lz4 is for the future PR?

Reply:
Yes, I thought I had removed this here already. I will remove it.